packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButtonContainer.js

```diff
@@ -25,29 +29,51 @@ function storeMapper(stores) {
 @observer
 class DoneAndTalkButtonContainer extends React.Component {
   render () {
-    const { 
+    const {
       completed,
       demoMode,
       disabled,
       goldStandardMode,
-      onClick, 
-      project, 
-      shouldWeShowDoneAndTalkButton, 
-      subject 
+      onClick,
+      project,
+      setOnHide,
+      shouldWeShowDoneAndTalkButton,
+      subject
     } = this.props
     const projectSlug = project && project.slug
     const subjectId = subject && subject.id
 
     if (shouldWeShowDoneAndTalkButton && projectSlug && subjectId) {
       const talkURL = `/projects/${projectSlug}/talk/subjects/${subjectId}`
 
+      function openTalkLinkAndClick (event) {
+        const isCmdClick = event.metaKey
+
+        onClick(event)
+          .then(() => {
+            if (window && talkURL) {
+              const url = `${window.location.origin}${talkURL}`
+              if (isCmdClick) {
+                event.preventDefault()
+                const newTab = window.open()
```

Comment:
This could also move into a callback, so that we don’t miss feedback by opening a new tab.

Reply:
Deferring this until after feedback closes breaks the classifier. The subject never advances and I'm not sure why.

Reply:
Fixed in the latest commit, which appends this code to the existing `onHide` callback, rather than overwriting it.